src/app/listing/listing.page.ts

```diff
@@ -75,4 +75,8 @@ export class ListingPage implements OnInit {
   goBack() {
     this.navCtrl.back();
   }
+
+  navigateToProject(data:any){
+    this.router.navigate(['project-details'], { state: data });
```

Comment:
will we always have state data here? or is there any possibility of getting other kind of data? can we generalize this key?

Reply:
this is only one where this state data will e used